loadtest/control/simulcontroller/controller.go

```diff
@@ -192,6 +192,11 @@ func getActionList(c *SimulController) []userAction {
 			run:       control.CreatePersistentNotificationPost,
 			frequency: 0.05,
 		},
+		{
+			name:      "WSReconnect",
+			run:       c.reconnectWebSocket,
```

Comment:
Nit just for consistency: the other actions use the name of the function as the `name` field.
```suggestion
			name:      "ReconnectWebSocket",
			run:       c.reconnectWebSocket,
```